doc/en/fixture.rst

```diff
@@ -34,6 +34,74 @@ both styles, moving incrementally from classic to new style, as you
 prefer.  You can also start out from existing :ref:`unittest.TestCase
 style <unittest.TestCase>` or :ref:`nose based <nosestyle>` projects.
 
+:ref:`Fixtures <fixtures-api>` are defined using the
+:ref:`@pytest.fixture <pytest.fixture-api>` decorator, :ref:`described
+below <funcargs>`. Pytest has useful built-in fixtures, listed here
+for reference:
+
+   :ref:`capfd-api`
+        Capture, as text, output to file descriptors ``1`` and ``2``
+
+   :ref:`capfdbinary-api`
+        Capture, as bytes, output to file descriptors ``1`` and ``2``
+
+   :ref:`caplog-api`
+        Control logging and access log entries
+
+   :ref:`capsys-api`
+        Capture, as text, output to ``sys.stdout`` and ``sys.stderr``
+
+   :ref:`capsysbinary-api`
+        Capture, as bytes, output to ``sys.stdout`` and ``sys.stderr``
+
+   :ref:`cache-api`
+        Store and retrieve values across pytest runs
+
+   :ref:`doctest_namespace-api`
+        Provide a dict injected into the docstests namespace
+
+   :ref:`monkeypatch-api`
+       Temporarily modify classes, functions, dictionaries,
+       ``os.environ``, and other objects
+
+   :ref:`pytestconfig-api`
+        Access to configuration values, pluginmanager and plugin hooks
+
+   :ref:`record_property-api`
+       Add extra properties to the test
+
+   :ref:`record_testsuite_property-api`
+       Add extra properties to the test suite
+
+   :ref:`recwarn-api`
+        Record warnings emitted by test functions
+
+   :ref:`request-api`
+       Provide information on the executing test function
+
+   :ref:`testdir`
+        Provide a temporary test directory to aid in running, and
+        testing, pytest plugins
+
+   :ref:`tmp_path-api`
+       Provide a :class:`pathlib.Path` object to a temporary directory
+       which is unique to each test function
+
+   :ref:`tmp_path_factory-api`
+        Make session-scoped temporary directories and return
+        :class:`pathlib.Path` objects
+
+   :ref:`tmpdir-api`
+        Provide a :class:`py.path.local` object to a temporary
+        directory which is unique to each test function;
+        replaced by :doc:`tmp_path <tmpdir>`
+
+        .. _`py.path.local`: https://py.readthedocs.io/en/latest/path.html
```

Comment:
We could add py to the intersphinx config, but maybe not worth it?  (it's meant to be phased out)